### ISSUE TITLE
StaticColor causes problems with nx-libs

### ISSUE THREAD

**Issue by mikedep333** [ArcticaProject/vcxsrv#4]
This bug corresponds to X2Go Bug 891.

By default, VcXsrv (both upstream and the X2Go/Arctica builds) offers 4 Visual Types with StaticColor:
0x211
0x212
0x213
0x214

Cygwin XWin and most Linux Xorg servers do not offer any Visual Types with StaticColor anymore (based on my limited research.)

This is a problem when VcXsrv is used with X2Go, or other nx-libs based solutions, because it causes nxagent to offer 1 Visual Type with StaticColor:
0xa6
When you suspend an nxagent session (x2go session), and try to resume it with a client (X server + nxproxy) that doe not support StaticColor, it fails to resume.

The default mode of VcXsrv is -wgl, which is affected. -swrastwgl is also affected. -nowgl is unaffected. So a workaround for many people is to launch VcXsrv with -nowgl.

VcXsrv is also unaffected when OpenGL acceleration is unavailable, such as over RDC or on a hypervisor with 3D support disabled. Presumably that causes VcXsrv to fallback to -nowgl mode.

**Comment by mikedep333** [comment]
As seen on the https://github.com/ArcticaProject/vcxsrv/tree/bugfix/bug4 branch, I have ported a patch from Cygwin XWin. I will discuss the implications for X2Go Client on the X2Go Bug report ([891](http://bugs.x2go.org/cgi-bin/bugreport.cgi?bug=891)). The patch disables any Visual Types where 3D acceleration would not work, which empirically prevents StaticColor from being offered. I tested it with X2Go successfully.
